generator.py

```diff
@@ -750,6 +751,8 @@ def get_options():
         default="data/professions.json",
         help="Data file for professions - defaults to %(default)s",
     )
+    parser.add_argument("-A", "--max-age", action="store", type=int, help="Maximum age of characters", default=55)
+    parser.add_argument("-a", "--min-age", action="store", type=int, help="Maximum age of characters", default=25)
```

Comment:
Help text needs a tweak.

Reply:
Damn it, I was working quickly on a bus but I thought I'd tidied up all my mistakes! :)